_posts/tsukori/2016-02-03-ch5exercises.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: post
+author: tsukori
+title: "Chapter 5: Loops Exercises"
+---
+
+Exercise 1: <iframe src="https://trinket.io/embed/python/b4be8e61d8" width="80%" height="450" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+Exercise 2: <iframe src="https://trinket.io/embed/python/46983a0b88" width="80%" height="450" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+
+Reflection: I had some trouble with the try/except blocks before but I think I have them down pretty well now. I also learned how while loops work in python, which game me some trouble while working on the clicky turtles exercise because my while loop would loop infinitely and freeze my browser. I want to start commenting my code more effectively so it's easier for people to read and make sense of it since we pair up for assignments. 
```

Comment:
That's a great aspiration- your partner will thank you :)

Another thing I'd suggest is to take a step back after your program is functional and see if the structure of the code makes the most sense.  For instance, in your programs, the last thing that happens is the answers are printed.  Yet that code is inside the while loop.  You could move that print statement outside the while loop and the functionality would be the same (since a break is happening anyways) but it becomes much more readable.  This kind of light refactoring will be really helpful as our programs get more complex.